docker-compose.yml

```diff
@@ -69,7 +71,7 @@ services:
     tty: true
     ports:
       - ${LOCAL_LARAVEL_PORT}:80
-      - "443:443"
```

Comment:
@jsur is this just your editor, or is there a reason we should be using single quotes vs double quotes?

Reply:
That seems to be Prettier using `.prettierrc.json` from the project root level:

```
["INFO" - 10:38:46 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 10:38:46 PM] Prettier Options:
{
  "filepath": "/Users/juliussuominen/code/coordinape/docker-compose.yml",
  "parser": "yaml",
  "singleQuote": true,
  "arrowParens": "avoid"
}
["INFO" - 10:38:46 PM] Formatting completed in 0.017ms.
```